.markdownlintrc

```diff
@@ -0,0 +1,5 @@
+{
+    "default": true,
+    "MD013": false,
+    "MD033": false
```

Comment:
This line disables the rule banning inline HTML: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md033---inline-html= but our ADRs use inline HTML

Reply:
Is it possible to add comments to these lines to explain what they do?

Reply:
great idea! Something felt off about explaining this in a PR comment rather than inline

<img width="741" alt="Screen Shot 2022-07-26 at 5 07 21 PM" src="https://user-images.githubusercontent.com/3699047/181112161-6513c563-fd5c-4e25-bde7-8a92cbe284f2.png">

Comments are possible in YAML so I'll switch this JSON file to YAML